riotfile.py

```diff
@@ -11,8 +11,9 @@
             venvs=[
                 Venv(
                     pkgs={
-                        "mypy": latest,
-                        "sphinx": latest,
+                        "mypy": "==0.961",
+                        "sphinx": "==5.1.1",
+                        "alabaster": "==0.7.12",
```

Comment:
I think we can probably remove the sphinx extension in another PR, since it is very ad-hoc for what we used to do for documentation. Otherwise, if we can think of some generic API, we could implement that instead.